cloud/disk_manager/internal/pkg/services/snapshots/create_snapshot_from_disk_task.go

```diff
@@ -372,3 +377,36 @@ func (t *createSnapshotFromDiskTask) GetResponse() proto.Message {
 		StorageSize: t.state.SnapshotStorageSize,
 	}
 }
+
+////////////////////////////////////////////////////////////////////////////////
+
+func (t *createSnapshotFromDiskTask) validateCheckpointStatus(
+	ctx context.Context,
+	nbsClient nbs.Client,
+	diskID string,
+	checkpointID string,
+) error {
+
+	status, err := nbsClient.GetCheckpointStatus(ctx, diskID, checkpointID)
```

Comment:
We need to ensure that NBS supports this call on every production cluster

Reply:
GetCheckpointStatus is not yet supported on all clusters.
Also, checkpoint creation should not fail when some node returned after maintenance with older version (where GetCheckpointStatus is unsupported)